src/screens/DomainScreen/elements/Header/index.js

```diff
@@ -186,8 +167,8 @@ const styles = StyleSheet.create({
   domainDescription: {
     fontFamily: fonts.inter[400],
     lineHeight: normalizeFontSize(17),
-    fontSize: normalizeFontSize(14)
-    // lineHeight: 17,
+    fontSize: normalizeFontSize(14),
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a color property to the `domainDescription` style is fine as long as it aligns with your design guidelines. However, please ensure that the `COLORS.white2` provides sufficient contrast against its background for readability.
```diff
- fontSize: normalizeFontSize(14)
+ fontSize: normalizeFontSize(14),
+ color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->